README.md

```diff
@@ -33,14 +33,22 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
+It's markup that introduces meaning to the web page.
 
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
+Display block does not allow HTML elements next to it unless by adding another element but display inline displays the elements inside the current block on the same line.
```

Comment:
Awesome!
check this for more comparison: https://www.w3schools.com/css/css_inline-block.asp